docs/route-tasks.md

```diff
@@ -0,0 +1,171 @@
+---
+id: route-tasks
+title: How to route Tasks
+sidebar_label: Route Tasks
+---
+
+Consider the specifications of a Worker:
+- A Worker only handles [Activity](docs/learn-activities) and [Workflow](docs/learn-workflows) Tasks that it is registered to handle. This means that a Task must be paired with a Worker that is registered to handle that Task.
+- A Worker runs on a specific host. This means that Activity Tasks that depend on host-specific resources, such as a local file, must be executed by a Worker on that host.
+
+The question is how do we make sure a specific Worker is executing a specific Task? We do this through "Task routing".
+
+## Task routing with Task Queues
+
+Task Queues are what enable "Task routing". Task routing is the development technique that pairs Tasks with Workers. The basics of it can be broken down into 3 steps:
+
+1. Create a Worker that is registered to handle Workflow and/or Activity Tasks.
+2. Subscribe the Worker to listen to a Task Queue.
+3. Configure Workflows and Activities to send Tasks to the Task Queue.
```

Comment:
It might be nice to either reference `ActivityOptions`/`WorkflowOptions` or say "configure activity execution"

Reply:
My goal with this was to explain what is happening, and then in the following example show how it happens. Will think about this -